readme.md

```diff
@@ -7,3 +7,7 @@ This repo contains the solution to this challenge https://www.notion.so/alexdan/
 The general idea is to have a navbar like the one in this image
 
 ![Navbar dribbble gif](https://cdn.dribbble.com/users/414694/screenshots/5487895/media/61c4a366fd91e33a29bdca6cfdf5f68d.gif)
+
```

Comment:
last todo: Let's rewrite this README to be a nice project for your resume. Suggestion:

- about: short description + demo link + screenshots (eg gif showing the animation)
- features: (the things you've done, dark theme, motion, reusable, storybook, cypress, etc)
- local setup: the steps on how to install, run, and test. (don't forget to document the Percy key!)

Reply:
the percy key should be an example key or can I put my own?

Reply:
nevermind 😆 `To enable Percy, the environment variable, PERCY_TOKEN, must be configured in your CI service. This is our write-only API token unique for each Percy project and should be kept secret.`
what do u mean with documenting the percy key, like how to create one or something?

Reply:
Ah, I see, if it's private, then pls don't paste it here. By documenting, a link pointing to the guide explaining how to create/use it is enough